frb_rust/src/ffi/web.rs

```diff
@@ -54,6 +54,28 @@ impl IntoDart for chrono::Duration {
         self.num_milliseconds().into_dart()
     }
 }
+
+#[cfg(feature = "uuid")]
+impl IntoDart for uuid::Uuid {
+    #[inline]
+    fn into_dart(self) -> DartAbi {
+        self.as_bytes().to_vec().into_dart()
+    }
+}
+
+#[cfg(feature = "uuid")]
+impl IntoDart for Vec<uuid::Uuid> {
+    #[inline]
+    fn into_dart(self) -> DartAbi {
+        use std::io::Write;
+        let mut buffer = Vec::<u8>::with_capacity(self.len() * 16);
+        for id in self {
+            let _ = buffer.write(id.as_bytes());
```

Comment:
nit: Not sure can we use `.map` and `.join`

Reply:
IMO looks fine, perhaps unwrap and fail fast instead of silently ignoring?

Reply:
this one [was already raised](https://github.com/sunshine-protocol/allo-isolate/pull/30#discussion_r977856082) by @shekohex upstream and matches with my implementation of `IntoDart` there : I settled down on not panicking as it raised a concern for him, and instead mentioned in the docs that it could return an incomplete buffer (guaranteed though, that any contained uuid will be valid) as per `std::io::Write::write`.